tests/Tests/200__monitor_and_manage/200__metrics/700_monitor.robot

```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Resource    ../../../Resources/ODS.robot
+Resource    ../../../Resources/Common.robot
+Resource    ../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Resource    ../../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library        JupyterLibrary
+Library        SeleniumLibrary
+Library        OpenShiftCLI
+Library        OperatingSystem
+
+
+*** Test Cases ***
+Verify that Prometheus image is a CPaaS built image with oc
+  [Tags]  ODS-734
+  ${pod} =  Run  oc get pods -n redhat-ods-monitoring -o json | jq '.items[] | select(.kind == "Pod") | select(.metadata.name | startswith("prometheus-")) | .metadata.name'
```

Comment:
You could create a "Search Pod" keyword for this in  tests/Resources/OCP.resource

Reply:
Thanks, Tarun it worked